Sources/Model/Swift/TracksLogging.swift

```diff
@@ -51,6 +51,10 @@ public class TracksLogging: NSObject, TracksLoggingConfiguration {
 
 private class DefaultTracksLogging: NSObject, TracksLoggingDelegate {
 
+    func logPanic(_ str: String) {
+        print("Tracks <☠️>: \(str)")
```

Comment:
Does it make sense for all platforms for logPanic to just call `func logMessageAndWait(_ message: String, properties: [String: Any]? = nil, level: SentryLevel = .info) throws`? If so, could this just directly call `func logMessageAndWait(_ message: String, properties: [String: Any]? = nil, level: SentryLevel = .info) throws` instead of having the platform do it?

Reply:
It would absolutely do. But the problem is... I tried, and it's not possible (or at least I've been unable!).

TracksLogging lives in a Swift Package that's not the same as the one wrapping up Sentry. Adding any kind of import for the `AutomatticRemoteLogging` package results in a compiler error (circular dependencies).

Reply:
Ahhhh, I see. Ugh, sorry, I see now why this is nasty....